src/components/Table/Table.tsx

```diff
@@ -65,7 +65,7 @@ export const Table = <RecordType extends GenericRecord>({
         footer={footer}
         loading={false}
         locale={intlLocale}
-        pagination={pagination}
+        pagination={{ ...Table.pagination, ...pagination }}
```

Comment:
Serve davvero? Nelle default props tanto le passiamo, se l'utente decide che non le vuole glielo lascerei fare?

Reply:
This is the fix requested in #291 . 

In this way, we allow to provide only a partial default pagination, overwriting only the necessary fields while maintaining the other default values. From the user's perspective, there is no need to spread Table.pagination explicitly.

Reply:
ok, I see that the code is changed 👍